### PR TITLE
Add iframe border and mobile styling

### PR DESCRIPTION
**Pull request by agrullon95**
#### Proposed Changes

* This PR adds styling changes for the launchpad.
I have modified the styling for the `Sidebar` and  `WebPreview` components to match the latest Figma designs (9wpoYJrMlYKAKUYvVBMN74-fi-150%3A3809). This includes both desktop and mobile styling.
* I have also added the new props, `showSiteAddressBar` & `devicesToShow` , to the `<WebPreview />` component from https://github.com/Automattic/wp-calypso/pull/66580 to hide the site preview address bar and the tablet icon on the device switcher above the site preview frame.

image

#### Testing Instructions

* Check out this PR branch
* Navigate to `http://calypso.localhost:3000/setup/launchpad?flow=newsletter&siteSlug=agrullon95.wordpress.com`
* Shrink/Expand the page as needed to view the launchpad page from a mobile & desktop perspective.

#### Pre-merge Checklist

- [ ] [Have you written new tests](https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md) for your changes?
- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [ ] Have you checked for TypeScript, React or other console errors?
- [ ] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?

Related to #66571, #66572, #66575

### CODE REVIEW

**Comment by jeyip** [comment]
## Testing
### Requirements
- [x] User can only switch between mobile / desktop previews
- [x] There is no site address bar in the preview
- [x] Mobile view matches figma mocks
- [ ] Desktop view matches mocks

### Browsers
- [x] Edge
- [x] Firefox
- [x] Chrome

**Comment by jeyip** [comment]
Go to admin button is cut off from the viewport and user has to scroll to see it. This is fine to handle in a follow up if you'd prefer. Below, the screen resolution is 1366 x 768

Screen Shot 2022-08-18 at 11 00 12 AM

File: client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/style.scss [hunk 1]
```diff
@@ -10,7 +10,7 @@
 		display: none;
 	}
 	.step-container {
-		background: var( --studio-gray-0 );
+		background: #fdfdfd;
```
Comment (jeyip) [inline client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/style.scss:13]: Was there a reason we switched from a css variable to a hard coded hex code?

File: client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/style.scss [hunk 2]
```diff
@@ -71,19 +102,19 @@
 }
 
 .launchpad__url-box {
-	background: var( --studio-white );
+	background: #f6f7f7;
```
Comment (jeyip) [inline client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/style.scss:105]: Same question as above https://github.com/Automattic/wp-calypso/pull/66614/files#r949443142

**Review by jeyip — APPROVED** [review]
🚢

**Comment by agrullon95** [comment]
> Go to admin button is cut off from the viewport and user has to scroll to see it. This is fine to handle in a follow up if you'd prefer. Below, the screen resolution is 1366 x 768

@nuriapenya had similar concerns with the admin link not being visible with certain screen sizes (height specifically) and is looking into this.

File: client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/style.scss (hunk 2 above)
Comment (agrullon95) [reply client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/style.scss:105]: I forgot to convert this. I will change `#f6f7f7` to `var(--studio-gray-0)`.

File: client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/style.scss (hunk 1 above)
Comment (agrullon95) [reply client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/style.scss:13]: I couldn't find a css variable for this hex code, so I didn't convert it to a variable.

**Pull request merged**